plugins/env/app/views/samson_env/_project_form.html.erb

```diff
@@ -20,6 +20,12 @@
   <% sorted_env_vars << EnvironmentVariable.new %>
   <%= render "samson_env/environment_variables", form: form, environment_variables: sorted_env_vars %>
 
+  <h4>
```

Comment:
only show that section if the bucket or help-text is set, most users don't want this feature I'd guess